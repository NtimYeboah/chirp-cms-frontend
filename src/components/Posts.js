/**
 * Created by ntimobedyeboah on 4/14/17.
 */
import React, { Component } from 'react';

class Posts extends Component {
    render() {
        return (
            <div>
                <h1>Post component</h1>
            </div>
        )
    }
}

module.exports = Posts;