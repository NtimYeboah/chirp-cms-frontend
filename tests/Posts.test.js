/**
 * Created by ntimobedyeboah on 4/15/17.
 */
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Posts from './../src/components/Posts';

describe('Post snapshot', () => {
    test('It should render posts component and take a snapshot', () => {
        const component = renderer.create(<Posts />);

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

describe('Post rendered output', () => {
    test('It should render Post component text', () => {
        const posts = shallow(<Posts />);

        expect(posts.find('h1').text()).toEqual('Post component');
    });
});

