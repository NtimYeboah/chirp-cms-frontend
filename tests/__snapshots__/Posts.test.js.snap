exports[`Post snapshot It should render posts component and take a snapshot 1`] = `
<div>
  <h1>
    Post component
  </h1>
</div>
`;
